discovery/syncer.go

```diff
@@ -154,7 +155,7 @@ func (s syncerState) String() string {
 const (
 	// DefaultMaxUndelayedQueryReplies specifies how many gossip queries we
 	// will respond to immediately before starting to delay responses.
-	DefaultMaxUndelayedQueryReplies = 10
+	DefaultMaxUndelayedQueryReplies = 50
```

Comment:
This can be removed now

Reply:
Done. 

Part of me wonders if we should keep a base line msg/s second limit _in addition_ to the bandwidth limit.

Reply:
Just thinking about a degenerate case where someone is having us send super small messages, so while we're under the total limit, we're doing more work overall.

Reply:
I think this only matters for messages we send out not ones that they send us, so the size of our sent messages is _less_ under their control (still under their control somewhat since they are querying).

Reply:
I think it'd be wise to also limit the number of messages so that they don't potentially sit in memory in the case of routing dumps, but this _should_ be addressed by the existing syncer semaphore.